stage-0-proposals.md

```diff
@@ -11,8 +11,10 @@ Stage 0 proposals are either
 | Annex B — HTML Attribute Event Handlers                            | Allen Wirfs-Brock                     | Allen Wirfs-Brock                     |                             |
 | [Array.isTemplateObject][isTemplateObject]                         | Mike Samuel                           | Mike Samuel                               |                             |
 | [Defensible Classes][defensible-classes]                           | Mark Miller<br />Doug Crockford       | Mark Miller<br />Doug Crockford       |                             |
+| [evalable][]                                                       | Mike Samuel                           | Mike Samuel                            |                             |
 | [Function bind syntax][bind-syntax]                                | Kevin Smith                           | Brian Terlson<br />Matthew Podwysocki | [March 2015][bind-notes]    |
 | [Function expression decorators][func-expr-decorators]             | Igor Minar                            | Igor Minar                            |                             |
+| [HostEnsureCanCompileStrings passthrough][heccspt]                 | Mike Samuel                           | Mike Samuel                               |                             |
```

Comment:
This one seems like a needs consensus PR more than a proposal to me.

Reply:
Ok.  So is the process:

1. rework it as a PR against tc39/ecma262/spec.html,
1. attach the "needs_consensus" label,
1. schedule 15 minutes under "9. Web compatibility issues / Needs Consensus PRs" on the agenda

Reply:
It's certainly up to you if you want to bring it as a proposal - it's possible that some delegates will insist it be one, but it's also possible that delegates will consider it more of a PR.

Reply:
In PR form: https://github.com/tc39/ecma262/pull/1498

Reply:
@ljharb Thanks for explaining.  I'll go with PR for now, and have the proposal ready if it's clear that's the way the committee wants to handle it.